packages/schematics/angular/migrations/update-14/update-tsconfig-target.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule, Tree } from '@angular-devkit/schematics';
+import { JSONFile } from '../../utility/json-file';
+import { getWorkspace } from '../../utility/workspace';
+import { Builders } from '../../utility/workspace-models';
+
+const SUPPORTED_BUILDERS = [Builders.Karma, Builders.NgPackagr, Builders.Browser];
+
+/** Migration to update tsconfig compilation target option to es2020. */
+export default function (): Rule {
+  return async (host) => {
+    // Workspace level tsconfig
+    updateTargetToES2020(host, 'tsconfig.json');
+
+    // Find all tsconfig which are refereced by the builders
+    const workspace = await getWorkspace(host);
+    for (const [, project] of workspace.projects) {
+      for (const [, target] of project.targets) {
```

Comment:
Nit: Could we do `for (const project of Object.values(workspace.projects)) { /* ... */ }`?

That would avoid the awkward destructure.

Reply:
Replaced it with;

```ts
    for (const project of workspace.projects.values()) {
      for (const target of project.targets.values()) {
```